README.md

```diff
@@ -52,26 +52,33 @@ where app.json may look something like:
             "sampleInterval": 1000
         }
     },
-    "plugins": {
-        "good": {
-            "opsInterval": 5000,
-            "requestHeaders": true,
-            "reporters": [{
-                "reporter": "good-console",
-                "events": { "response": "*", "ops": "*", "log": "*", "error": "*" }
-            },
-            {
-                "reporter": "good-file",
-                "events": { "response": "*", "error": "*" },
-                "config": "/log/response.log"
-            },
-            {
-                "reporter": "good-file",
-                "events": { "ops": "*" },
-                "config": "/log/ops.log"
-            }]
+    "registrations": [
```

Comment:
hapijs/glue#31 explains the change to the manifest format.

The simplest form you can pass as registrations is:
```javascript
"registrations": [
  { "plugin": "good" },
  { "plugin": "tv" }
]
```

I'd possibly be open to a pull request to support the form of:

```javascript
"registrations": [
   "good",
   "tv"
]
```

But I'm not sold on it yet.